terraform/modules/matrix-service/infra/converter_batch.tf

```diff
@@ -108,7 +108,7 @@ resource "aws_batch_compute_environment" "converter_compute_env" {
     // You must set desired_vcpus otherwise you get error: "desiredvCpus should be between minvCpus and maxvCpus"
     // However this is actually not settable in AWS.  It will not let you change it.
     // Here we use an external data source to dynamically set the desired vcpus to match current state.
-    desired_vcpus = "${data.external.converter_desired_vcpus.result.desired_vcpus}"
+    desired_vcpus = 4
```

Comment:
Why this change? This will cause terraform to fail when the set desired vcpus does not match 4? We should keep the external data source reference

Reply:
Woops meant to change that back. That was a workaround since the script wasn't working for me.

Reply:
Will investigate script after prod is green